xenit/build.gradle

```diff
@@ -0,0 +1,51 @@
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile.From
+
+subprojects {
+
+    configurations {
+        telemetry
+    }
+
+    ext {
+        alfredTelemetryVersion = '0.4.0'
+    }
+
+    apply plugin: 'java'
+    apply plugin: 'eu.xenit.docker'
+    apply from: "${project.projectDir}/overload.gradle"
+
+
+    def upstreamProjectName = ":upstream:${project.name}"
+    def upstreamProject = project(upstreamProjectName)
+    evaluationDependsOn(upstreamProjectName)
+    def upstreamImage = upstreamProject.getTasks().getByName('buildDockerImage')
+    buildDockerImage.dependsOn(upstreamImage)
+    createDockerFile.dependsOn(upstreamImage)
+
+    buildDockerImage {
+        pull = false
+    }
+
+    dockerFile {
+        dockerBuild {
+            repository = [calcRepository(upstreamProject.solr.flavor,true)].get(0)
+            tags = calcTags(upstreamProject.solr.version)
+        }
+    }
+
+    createDockerFile {
+       from(upstreamImage.getImageId().map({ new From(it) }))
+        // micrometer handler init script
+        smartCopy("${project.projectDir}/../local/95-init-solr-micrometer-metrics.sh","/docker-entrypoint.d/")
+        if("solr4".equals(upstreamProject.solr.flavor)) {
```

Comment:
instead of using the `if` statement here, I would move the `createDockerfile` in the respective subprojects build.gradle, but that's maybe just a personal preference

Reply:
Atm subprojects have no build.gradle. 
Should more differences appear between flavors, we might think about creating new build.gradle